scanner/config.go

```diff
@@ -91,7 +93,12 @@ func Config(searchDir string) models.ScanResultModel {
 	// Setup
 	currentDir, err := os.Getwd()
 	if err != nil {
-		result.AddError("general", fmt.Sprintf("Failed to expand current directory path, error: %s", err))
+		result.AddErrorWithRecommendation("general", models.ErrorWithRecommendations{
+			Error: fmt.Sprintf("Failed to expand current directory path: %s", err),
```

Comment:
move `fmt.Sprintf("Failed to expand current directory path: %s", err)` into a var to avoid code duplication